app/presenters/provider/admin/user/access_tokens_new_presenter.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Provider::Admin::User::AccessTokensNewPresenter
+
+  def initialize(provider)
+    @provider = provider
+    @timezone = ActiveSupport::TimeZone.new(provider.timezone)
```

Comment:
I don't think it matters here but I wouldn't expose provider
```suggestion
    @timezone = ActiveSupport::TimeZone.new(provider.timezone)
```

Reply:
https://github.com/3scale/porta/pull/3943/commits/5ada2fa500a9868bca1a7783bffb5c1d55d2c3a3